src/Xamarin.Android.Build.Tasks/Xamarin.Android.Common.targets

```diff
@@ -120,11 +120,11 @@ Copyright (C) 2011-2012 Xamarin. All rights reserved.
 <!--
 *******************************************
   Extensibility hook that allows VS to
-  provide extra behavior without modifying 
+  provide extra behavior without modifying
```

Comment:
Is there any way to *avoid* all of these whitespace changes?

Sure, in theory, they improve the code base.  But it's making this PR *really* noisy.

Reply:
The `*.targets` files are often badly formatted and my editor removes trailing whitespace (as I want it to). I'll hand-edit the changes.